docs/docs/high-availability.md

```diff
@@ -6,14 +6,13 @@ title: High Availability
 
 ## Introduction
 
-Marathon by default supports a high availability mode of operation which allows applications to continue running if an individual instance becomes unavailable. This is accomplished by running several Marathon instances pointing to the same ZooKeeper quorum. ZooKeeper is used to perform leader election in the event that the currently leading Marathon instance fails.
-
+Marathon supports a high availability mode of operation by default. High availability mode allows applications to continue running if an individual instance becomes unavailable. This is accomplished by running several Marathon instances that point to the same ZooKeeper quorum. ZooKeeper is used to perform leader election in the event that the currently leading Marathon instance fails.
 
 ## Configuration
 
-When set to `true`, the `--ha` command line argument launches Marathon in high availability mode. `true` is the default, so it does not need to explicitly be set.
+When set to `true`, the `--ha` command line argument launches Marathon in high availability mode. `true` is the default setting.
```

Comment:
This changed in Marathon v0.13.

The following flags are valid: `--[disable_]ha` (Optional. Default: enabled)

That means that you can disable HA mode if you pass `--disable_ha`, otherwise Marathon will run in HA mode.